queries/haskell/highlights.scm

```diff
@@ -190,26 +176,38 @@
          ]))
      (quasiquote)
      ((exp_name) . (operator))
-    ]))
+    ])+
```

Comment:
Do we still need to quantify `(function)` here ? I don't think it's that helpful anymore considering we're limiting it to the function right after its signature